src/components/Inputs/index.tsx

```diff
@@ -96,7 +97,17 @@ export const InputErrorMessage = styled(motion.label)<InputProps>`
   color: ${({ theme }) => theme.global.alert};
 `
 
-export const InputLabel = styled(motion.label)`
+export let InputLabel: FC<HTMLMotionProps<'label'> & { inputHasValue: boolean }> = ({ inputHasValue, ...props }) => (
```

Comment:
Are children being passed via props? Didn't know you could do that...!

Reply:
Yay :)